torch_geometric/transforms/to_sparse_tensor.py

```diff
@@ -4,39 +4,57 @@
 from torch_geometric.data.datapipes import functional_transform
 from torch_geometric.transforms import BaseTransform
 from torch_geometric.typing import SparseTensor
-from torch_geometric.utils import sort_edge_index
+from torch_geometric.utils import sort_edge_index, to_torch_coo_tensor
 
 
 @functional_transform('to_sparse_tensor')
 class ToSparseTensor(BaseTransform):
     r"""Converts the :obj:`edge_index` attributes of a homogeneous or
     heterogeneous data object into a (transposed)
-    :class:`torch_sparse.SparseTensor` object with key :obj:`adj_t`
+    :class:`torch_sparse.SparseTensor` or a :class:`torch.spase.Tensor`
```

Comment:
```suggestion
    :class:`torch_sparse.SparseTensor` or a native PyTorch sparse tensor
```
Is `torch.sparse.Tensor ` a new type in pytorch? I thought Pytorch sparse tensor was of type `Tensor`.

Reply:
No. Actually `torch.sparse.Tensor` is an alias of `torch.Tensor`.